plugins/psv-extractor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@flatfile/plugin-psv-extractor",
-  "version": "0.0.3",
+  "version": "1.0.0",
```

Comment:
Feel like we should let the release CI/CD increment this as a Major instead of versioning here. Pretty sure the patch changeset will make this `1.0.1`.

Reply:
```suggestion
  "version": "0.0.3"
```

Reply:
Now that the changeset is `major`, this will bump to `"version": "2.0.0"` - is that the intention?